README.md

```diff
@@ -1,23 +1,19 @@
+[![Build status](https://github.com/navikt/syfoalerts/workflows/Deploy%20to%20dev%20and%20prod/badge.svg)](https://github.com/navikt/syfoalerts/workflows/Deploy%20to%20dev%20and%20prod/badge.svg)
+
 # syfoalerts
 
 Lager alerts for team barken sine syfo-apper
 
 For mer informasjon om hvordan alarmene fungere se:
 `https://github.com/nais/doc/tree/master/content/alerts`
 
-### For NAV ansatte
-Vi er tilgjenngelig på slack kanalen #barken
-
 ## Utvikling:
-En kan bruke `https://prometheus.nais.preprod.local/graph` som hjelp til å teste queries. Bruk `test/syfoalert_test.yaml` til å teste alerteren i dev. Varsel dukker da opp i `syfo-alerts-dev` på Slack
-
-### Deploy test:
-Deploy alerten din til riktig cluster i preprod:
-`kubectl apply -f test/syfoalert_test.yaml`
-
-Om alerten allerede finnes må den fjernes først:
-`kubectl delete alert syfoalerts-test`
+En kan bruke `https://prometheus.nais.preprod.local/graph` som hjelp til å teste queries.
 
 ### Deploy prod:
-Bruk byggjobb:
-`https://jenkins-digisyfo.adeo.no/job/syfoalerts/`
+Alle kode som er i master går til prod-fss
+Dette kan evt gjøres manuelt med følgende kommando:
+`kubectl apply --context prod-fss --namespace default -f sykmeldingalerts.yaml`
+
+### For NAV ansatte
+Vi er tilgjenngelig på slack kanalen #team-sykmelding
```

Comment:
Dette her er egentlig alerts for apper som ingen eier for øyeblikket, men som likevel er viktige for at totalen skal fungere 🤷‍♀